react-rendering-lists/my-app/src/PokemonList.js

```diff
@@ -0,0 +1,8 @@
+export default function PokemonList({ pokedexArray }) {
```

Comment:
From a stylistic standpoint, we don't usually put the type of the object in the variable name. It's usually clear from context (or documentation). So just `pokedex` would be a more typical name.

Reply:
Makes sense.  Thx Robert.